app/models/purchase.py

```diff
@@ -45,23 +45,54 @@ class Order(db.Model):
     date = db.Column(db.DateTime)
     status = db.Column(db.Integer)
 
-    def __init__(self, cart_items):
-        self.date = datetime.now(pytz.timezone('US/Eastern'))
+    vendor_id = db.Column(db.Integer)
+    company_name = db.Column(db.String(64))
+
+    def __init__(self, date, vendor_id):
         self.status = Status.PENDING
+        self.date = date
+        self.vendor_id = vendor_id
+        vendor = User.query.get(vendor_id)
+        self.company_name = vendor.company_name
+
+    def __repr__(self):
+        return "<Order: {}>".format(self.id)
+
+    @staticmethod
+    def order_cart_items_from_vendor(vendor_id, date=None):
+        """Orders all the items in the cart from a given vendor"""
+
+        if date is None:
+            date = datetime.now(pytz.timezone('US/Eastern'))
```

Comment:
Why keep this as Eastern time zone when the earlier definition of date on ln 45 is non formatted. See also ln 91